### ISSUE TITLE
Creating an analysis with GG2 tables

### ISSUE THREAD

**Issue by wasade** [qiita-spots/qiita#3393]
I created an analysis with a GG2 filtered table, and was surprised to see it described as a Greengenes 13_8 table. 

Is a user able to select artifacts that are GG2 filtered for analysis?

<img width="1668" alt="Screen Shot 2024-04-17 at 9 01 51 AM" src="https://github.com/qiita-spots/qiita/assets/474290/04b62f45-fb06-49e1-bfbc-014e72bcdab7">

**Comment by antgonza** [comment]
The Merging Scheme is: "parent command | grand parent command" and for those artifacts:
- parent is filter features from the gg2 plugin
- grandparent is Deblur 2021.09 filtered against GG13.8 as you are using the reference-hit tables

Thus, is accurate. A way to "solve" this would be to rename the commands in GG2 to reflect that, like: "Filter features against GG2 reference".

**Comment by wasade** [comment]
Thanks! It should note the version of GG2 as well. Should this be upstream of the merging scheme though, and part of the artifact information?

**Comment by antgonza** [comment]
That's right. Note that all the names are inherited from the original commands (in this case the GG2 Q2 plugin) so the changes need to happen there not in Qiita.

**Comment by wasade** [comment]
How does this differ conceptually from representing how the GG 13_8 database is described as used? Qiita defines and creates artifact information, right?

If this does not need to change in Qiita, do you know what needs to change and where?
